docs/inquiries/commercial-inquiries.md

```diff
@@ -0,0 +1,6 @@
+Commercial inquiries
+--------------------
+
+If you have any questions about our services feel free to contact us using
+[Dasharo contact form](https://dasharo.com/pages/contact.html) or 
+[3mdeb contact form](https://3mdeb.com/contact/).
```

Comment:
Maybe also copy a part of ending from my https://github.com/3mdeb/news-and-ideas/pull/398/files , in particular:
```
Feel free to [book a call with us](https://calendly.com/3mdeb/consulting-remote-meeting) or
drop us an e-mail at <contact@3mdeb.com>, and we will do our best to help your company
to reach new heights.
```

Reply:
@pietrushnic I did these two minor changes in https://github.com/Dasharo/docs/commit/f9c1427f6a2954be1ed30d1827a840abd6ead86d